orga/tests/acceptance/join_test.js

```diff
@@ -98,6 +98,15 @@ module('Acceptance | join', function (hooks) {
       assert.notOk(currentSession(this.application).get('isAuthenticated'), 'The user is still unauthenticated');
       assert.contains(this.intl.t('pages.login-form.invitation-was-cancelled'));
     });
+
+    test('it should redirect user to login page when user tries to change language by url', async function (assert) {
```

Comment:
```suggestion
    module('when user tries to change language by url', function () {
      test('redirects the user to login page', async function (assert) {
```

Reply:
J'ai eu du mal à comprendre le pourquoi de ce test. C'est le code `FAKE999` qui m'a donné l'indice que la campagne n'existait pas. Aussi il me semble qu'il faudrait améliorer le titre du test pour indiquer que même quand on met `lang` dans un URL d'une campagne qui n'existe pas on est redirigé vers la page de login.

Une suggestion : 

```suggestion
    module('when URL of a non-existent campaign', function () {
      module('when lang query param in URL', function () {
        test('redirects to login page', async function (assert) {
```

Reply:
Y'a une coquille @lego-technix 
J'ai ajouté ce test de caractérisation car j'ai observé qu'en ajoutant lang=en même sur une invitation valide, il y a une redirection vers connexion. 
J'ai pas creusé le pourquoi, je suppose que c'est la mécanique ember.
Ca m'a semblé utile néammoins de l'ajouter pour le refacto à venir quand on voudra supprimer l'utilisation de ?lang= pour la traduction
Je corrige en ajoutant une invitation valide

Reply:
@igorissen tous les tests sont sur le format "it should do something wen something happen", tu suggères de modifier tout le fichier de test en 

```
module('wen something happen")
test('does something')
```

Reply:
Bien reçu et je suis bien d'accord sur l'intérêt du test.

Si le pourquoi du comportement du test n'est pas maîtrisé par nous il faut faire apparaître cette incompréhension, sinon les personnes qui liront ce test après nous penserons que nous savions exactement ce que nous faisions. Et je suggère de mettre cette incertitude de compréhension dans le titre d'un `test` ou d'un `module` et non dans un commentaire de code pour que cette connaissance ne se perde pas.

Reply:
Pour moi on peut faire « Resolve conversation ». Et pour toi @igorissen ?

Reply:
Alors, je viens de comprendre dans quel cas on est, ça fait un GET sur http://localhost:4201/api/organization-invitations/100057?code=INVIABC123%3Flang
qui répond une 404
Not found organization-invitation for ID 100057 and code INVIABC123?lang
@er-lim

Reply:
Ma suggestion c'était uniquement sur les tests que tu ajoutes ou que tu modifies, de retirer le `it`. Ce n'est pas bloquant, donc si c'est plus simple de le faire à un autre moment pas de souci pour moi.